examples/timeline/editing/controlItems.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <title>Timeline | editing | Manipulation example</title>
```

Comment:
`Manipulation example` => `Control Item example`

Reply:
changed to Control Item example